.github/workflows/ci-linux.yml

```diff
@@ -66,124 +43,21 @@ jobs:
       targets_pre: all-sage-local
       targets: build doc-html
       targets_optional: ptest
+      tox_system_factors: >-
+          ["ubuntu-jammy"]
       tox_packages_factors: >-
-          ["standard"]
+          ["minimal"]
       docker_push_repository: ghcr.io/${{ github.repository }}/
-      # Make sure that all "standard" jobs can start simultaneously,
-      # so that runners are available by the time that "default" starts.
-      max_parallel: 50
-
-  standard-constraints_pkgs-norequirements:
-    if: ${{ success() || failure() }}
```

Comment:
Just turn this off. Please do not remove things broken. They may be fixed later.